lib/utils/other/webapp_manager.dart

```diff
@@ -1,18 +1,21 @@
 import 'dart:io';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:pangolin/utils/data/dap_index.dart';
 import 'package:pangolin/utils/data/app_list.dart';
-import 'package:pangolin/utils/data/models/application.dart';
+import 'package:pangolin/utils/data/dap_index.dart';
 
 void wapInstall(String url) {
   Process.run('wget', [
     "-N",
-    "https://packages.dahliaos.io/" + url,
-    "-P" "/root/Applications/data/"
+    "https://packages.dahliaos.io/$url",
+    "-P",
+    "/root/Applications/data/",
   ]).then((result) {
-    print(result.stderr.toString());
-    print(result.stdout.toString());
+    if (kDebugMode) {
+      print(result.stderr.toString());
+      print(result.stdout.toString());
+    }
```

Comment:
use debugPrint should be better than make a if check, same for kdebugMode checks

Reply:
I think the avoid_print rule should be disabled for the whole file